Learning-CSharp/Learning.DesignPatterns/ConsoleInput.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+
+namespace Learning.DesignPatterns
+{
+    internal class ConsoleInput
+    {
+        internal static bool TryReadDecimal(string v, out decimal expenseReportAmount)
+        {
+            expenseReportAmount = default;
```

Comment:
"default" literal isn't available in C#7.0 version. Please use "decimal.Zero" instead.

Reply:
Fixed by adding decimal.Zero instead of default.